### PR TITLE
uptick version to 3.7.10

### PR DESCRIPTION
**Pull request by junpuf**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
~~* [ ] Bumped the build number (if the version is unchanged)~~
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

#450

### CODE REVIEW

**Comment by junpuf** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ source:
       - patches/0009-runtime_library_dir_option-Use-1st-word-of-CC-as-com.patch
       - patches/0010-Win32-Do-not-download-externals.patch
       - patches/0011-Add-support-for-_CONDA_PYTHON_SYSCONFIGDATA_NAME-if-.patch
-      - patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
+      #- patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch
```
Comment (jakirkham) [inline recipe/meta.yaml:32]: Why is this patch commented?

**Comment by jakirkham** [comment]
cc @mingwandroid (for vis)

File: recipe/meta.yaml (hunk 1 above)
Comment (junpuf) [reply recipe/meta.yaml:32]: this patch needs some fix, i first removed it to try it out, and CI passed. Now I'm trying to see how to fix the 0012 patch. I think some lines needs to be updated.

Comment (junpuf) [reply recipe/meta.yaml:32]: looking at the 0012 patch file, it seems to be generated file. @jakirkham is there anything i should read about to fix this?

Comment (jakirkham) [reply recipe/meta.yaml]: It's probably something simple like more code was added to the file so the line numbers are off. Basically this if just trying to prioritize to search for libraries to load in the Conda environment. Though I could be wrong about this

This was Ray's patch. Have already cc'd him. Would give him a chance to respond

File: recipe/patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch [hunk 2]
```diff
@@ -54,7 +54,7 @@ index 97973bc..1340ad7 100644
          def find_library(name):
              # See issue #9998
 -            return _findSoname_ldconfig(name) or \
--                   _get_soname(_findLib_gcc(name) or _findLib_ld(name))
+-                   _get_soname(_findLib_gcc(name)) or _get_soname(_findLib_ld(name))
```
Comment (junpuf) [inline recipe/patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch:57]: this may have some change, I'll figure this out.

Comment (isuruf) [reply recipe/patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch:57]: Have a look at 3.8 branch's patch. That might help. https://github.com/conda-forge/python-feedstock/blob/3.8/recipe/patches/0013-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch

Comment (junpuf) [reply recipe/patches/0012-Fix-find_library-so-that-it-looks-in-sys.prefix-lib-.patch:57]: Yeah, that confirmed it. I think this is fine.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -70,7 +70,7 @@ source:
     sha256: de3c87b26a80e789986d8e6950c6304175d3829afe9c6c7211eb7257266ab0ac         # [win]
 
 build:
-  number: 100
+  number: 0
```
Comment (isuruf) [inline recipe/meta.yaml:73]:
```suggestion
  number: 100
```

Comment (jakirkham) [reply recipe/meta.yaml]: Why do we need this build number Isuru?

Comment (isuruf) [reply recipe/meta.yaml]: To fix some issues where pypy is prioritised.

Comment (jakirkham) [reply recipe/meta.yaml]: Ah ok. Do we need this for Python 3.6 as well?

**Pull request merged**